feature-extensions/apigee/jazz-apigee-proxy/jazz-apigee-proxy.js

```diff
@@ -0,0 +1,97 @@
+// =========================================================================
```

Comment:
why is this here. This is something already added to core. We need to discuss this

Reply:
As per the PR description, this is more of an initial take on it. We can discuss and figure out the best location for the code related to these plugins/extensions.

``Figuring out a better place to keep @svidhani's apigee gateway function, I am not advocating that we put it here, just did that for convenience.``

Reply:
makes sense. we should have a discussion on this